test/docgen/view/documentation.test.ts

```diff
@@ -10,7 +10,7 @@ const LIBRARIES = `${__dirname}/../../__fixtures__/libraries`;
 
 // this is a little concerning...we should be mindful
 // if need to keep increasing this.
-jest.setTimeout(2 * 60 * 1000);
+jest.setTimeout(3 * 60 * 1000);
```

Comment:
The new test I added takes more than 2 minutes... :\

Reply:
Ouch